app/charts/combo/chart-combo-line-single.tsx

```diff
@@ -97,7 +97,13 @@ export const ChartComboLineSingle = React.memo(
           <Tooltip type="multiple" />
         </ChartContainer>
         <ChartControlsContainer>
-          <LegendColor chartConfig={chartConfig} symbol="line" />
+          <LegendColor
+            chartConfig={chartConfig}
+            symbol="line"
+            getLegendItemDimension={(label) => {
+              return measures.find((measure) => measure.label === label);
+            }}
```

Comment:
Should this be memoized? Im assuming that the this will be re-rendered with every change to the chart or config?

Reply:
You mean `getLegendItemDimension` (with `useCallback`)? I can add this 👍 The `LegendColor` function looks to be already memoized 👀